addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CrossSiteScriptingScanRule.java

```diff
@@ -310,6 +310,22 @@ private boolean performTagAttack(
 
             for (HtmlContext ctx : contexts2) {
                 if (ctx.isInUrlAttribute()) {
+
+                    // No payload modifications should be made in case of "javascript:" protocol
+                    // attacks
+                    String attAndPayload =
+                            ctx.getTagAttribute()
```

Comment:
You need to have in consideration that between the tag attribute name, equals, and value you may have white space characters like : `<iframe src = "javascript:alert(1)">` . You can modify s`shouldReportXssInFrameSrcTag()` test to try it.

Reply:
Correct, will make the necessary changes.

Reply:
Same here.